image/generic/sle16/config.sh

```diff
@@ -0,0 +1,93 @@
+#!/bin/bash
+#================
+# FILE          : config.sh
+#----------------
+# PROJECT       : KIWI - Image System
+# COPYRIGHT     : (c) 2023 SUSE Software Solutions Germany GmbH
+#               :
+# AUTHORS       : Marcus Schaefer <ms@suse.de>
+#               : Keith Berger <kberger@suse.com>
+#               :
+# BELONGS TO    : Operating System images
+#               :
+# DESCRIPTION   : configuration script for SUSE based
+#               : operating systems
+#----------------
+#======================================
+# Functions...
+#--------------------------------------
+test -f /.kconfig && . /.kconfig
+test -f /.profile && . /.profile
+
+#======================================
+# Greeting...
+#--------------------------------------
+echo "Configure image: [$kiwi_iname]..."
+
+#======================================
+# Setup baseproduct link
+#--------------------------------------
+suseSetupProduct
+
+#======================================
+# Setup default target, multi-user
+#--------------------------------------
+baseSetRunlevel 3
+
+#======================================
+# Deactivate services
+#--------------------------------------
+# cloud-regionsrv-client packages(s) activates that service
+# on install. For migration we don't need this service.
+# The installation of that packages and its relatives happens
+# to satisfy the dependencies on the zypper plugin side but
+# not for actually registering a guest to the public cloud
+# infrastructure. We expect that step to be already done
+suseRemoveService guestregister
```

Comment:
Please drop the use of suseRemoveService, it's deprecated and will be removed in newer versions of kiwi

On a general advice, do not use any of the functions that we provide via `test -f /.kconfig && . /.kconfig` We want to get rid of any of those `suseXXX` methods. 

In the above case call systemctl directly